tests/phpunit/api/v3/AttachmentTest.php

```diff
@@ -349,9 +349,6 @@ public function testCreate($testEntityClass, $createParams, $expectedContent) {
       'return' => ['content'],
     ]);
     $this->assertEquals($expectedContent, $getResult2['values'][$fileId]['content']);
-    foreach (['id', 'entity_table', 'entity_id', 'url'] as $field) {
-      $this->assertEquals($createResult['values'][$fileId][$field], $getResult['values'][$fileId][$field], "Expect field $field to match");
-    }
```

Comment:
Hmm, yeah, I think the intent was to ensure that the `get` always returned essential fields (`id`, `entity_table`, `entity_id`) even if the `return` list didn't specify them, and the old snippet looks like a copy-paste error. (`s/$getResult/$getResult2/`)

Good point, though, that checking `$getResult` is effectively superfluous here.

Reply:
Ah yes I see it does return those even if you only ask for `content`. So I can put it back replacing with the getResult2 and removing url, and then adding a comment that that's why it's there.

Reply:
@demeritcowboy maybe replicate it what we did with the url above as well